### ISSUE TITLE
Ability to set specific time

### ISSUE THREAD

**Issue by Rafii2198** [elijahjibben/Wynncraft-Dynamic-Weather#2]
It is technically not what the is about, but I think it would be really nice if there was an option to make it so for example in The Silent Expanse or The Void it would be permanently nighttime. It is mainly for shader users are many shaders do not look good when the End biome is applied in the overwold, in terms of Complementary shaders which are really popular during the day everything is so dark it is super hard to see. Nighttime in these area works well, so if the mod could make it when you are there the time would be changed to Night would completely fix this issue

**Comment by elijahjibben** [comment]
That is a really good suggestion, I was just noticing this yesterday in the Qira Hive too. This will be in the next feature release, along with (hopefully) a built-in particle rendering system that gets rid of the need for Particle Rain and lets me make fully custom weather types for any region.

**Comment by Rafii2198** [comment]
Yeah every time they use The End biome it is really hard to see anything during the day, that also includes the Roots of Corruption, so probably the transition between these 2 places won't look that good but I think having ability to see anything will be a lot better, other places are pretty separated from the rest of terrain so it won't be an issue.
And thanks for accepting the suggestion! :)

**Comment by elijahjibben** [comment]
Good point on transitions, come to think of it I've seen some mods do kind of an "animation" where it increments the visible time over 5s until the desired time so it would be more of a transition rather than jump. I'll look into that!
